ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml

```diff
@@ -1881,6 +1881,23 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-baremetalds-ipi-ovn-ipv4
+- as: baremetal-upi-ovn-ipv4-p1-f14
+  cluster: build05
+  cron: 30 21 7,21 * *
+  steps:
+    cluster_profile: equinix-ocp-metal-qe
+    env:
+      AUX_HOST: openshift-qe-bastion.arm.eng.rdu2.redhat.com
+      DISCONNECTED: "false"
```

Comment:
`DISCONNECTED` defaults to "false", so we can skip it there.

Reply:
done, thanks